File: strings/python/data1.py
class DateData:
    def __init__(self, day, month, year):
        self.day = day
        self.month = month
        self.year = year


def extract_date_data(date):
    parts = date.split('/')
    day = int(parts[0])
    month = int(parts[1])
    year = int(parts[2])
    return DateData(day, month, year)


obj = extract_date_data("21/07/2010")
print(f"Dia: {obj.day}")
print(f"Mês: {obj.month}")
print(f"Ano: {obj.year}")

File: strings/java/src/Data1.java
public class Data1 {

	public static DateData extractDateData(String date) {

		String[] parts = date.split("/");

		int day = Integer.parseInt(parts[0]);
		int month = Integer.parseInt(parts[1]);
		int year = Integer.parseInt(parts[2]);

		return new DateData(day, month, year);
	}

	public static void main(String[] args) {
		DateData obj = extractDateData("21/07/2010");
		System.out.println("Dia: " + obj.day);
		System.out.println("Mês: " + obj.month);
		System.out.println("Ano: " + obj.year);
	}
}

class DateData {
	int day;
	int month;
	int year;

	public DateData(int day, int month, int year) {
		this.day = day;
		this.month = month;
		this.year = year;
	}
}

